.github/workflows/dependency-check.yml

```
@@ -1,33 +1,21 @@
 
-  push:
-    paths-ignore:
-      - 'docs/**'
-      - 'README.md'
-      - 'CHANGELOG.md'
-      - '.gitignore'
-      - '.gitattributes'
-      - 'infection.json.dist'
-      - 'phpunit.xml.dist'
-      - 'psalm.xml'
+  push: *ignore-paths
```

actionlint (1.7.9)

"push" section is alias node but mapping node is expected

(syntax-check)